### PR TITLE
Do some restructuring of Keras event

### PR DESCRIPTION
**Pull request by osanseviero**
There are still some TODOs, @merveenoyan woudl you like to take them?

### CODE REVIEW

File: keras-dreambooth-sprint/README.md [hunk 1]
```diff
@@ -1,95 +1,135 @@
...
-If you need more pointers on Dreambooth implementation with Keras, you can check out [this repository](https://github.com/sayakpaul/dreambooth-keras). 
+## Compute
 
 [ADD DETAILS ABOUT COMPUTE SPONSORSHIP]
 
-### Dreambooth Diffusers Integration with KerasCV
+## Model Training
 
-As of now, inference and deployment options of `KerasCV` is limited, which `diffusers` library could come for a rescue. With only few lines of code, we can convert a `KerasCV` model into a `diffusers` one and use `diffusers`’ pipelines to perform inference. You can get more information [here](https://huggingface.co/docs/diffusers/main/en/using-diffusers/kerascv). Also check out [this Space](https://huggingface.co/spaces/sayakpaul/convert-kerascv-sd-diffusers) for converting your `KerasCV` model to `diffusers`one.
+You can find the notebook here and adjust it according to your own dataset 👇
```
Comment (merveenoyan) [inline keras-dreambooth-sprint/README.md:71]: I'd love it if you could discard this change and keep the guide informative.

File: keras-dreambooth-sprint/README.md [hunk 2]
```diff
@@ -1,95 +1,135 @@
...
 
-```
-from diffusers import StableDiffusionPipeline
+TODO: The notebook below is a Google Drive. We should add it to the GitHub repo
 
-# checkpoint of the converted Stable Diffusion from KerasCV
-model_ckpt = "sayakpaul/text-unet-dogs-kerascv_sd_diffusers_pipeline"
-pipeline = StableDiffusionPipeline.from_pretrained(model_ckpt)
-pipeline.to("cuda")
+[Link to notebook](https://colab.research.google.com/drive/1Bh49YBYgaNc3X5aRzaMZNska4ErvU3lJ?usp=sharing). If you need more pointers on Dreambooth implementation with Keras, you can check out [this repository](https://github.com/sayakpaul/dreambooth-keras).
```
Comment (merveenoyan) [inline keras-dreambooth-sprint/README.md:75]: I'd love it if you could discard this change and keep the guide informative.

I don't want people to jump to other places.

File: keras-dreambooth-sprint/README.md [hunk 3]
```diff
@@ -1,95 +1,135 @@
...
 
-unique_id = "sks"
-class_label = "dog"
-prompt = f"A photo of {unique_id} {class_label} in a bucket"
-image = pipeline(prompt, num_inference_steps=50).images[0]
-```
+Note/q: having a colab and then a repo could maybe be confusing?
```
Comment (merveenoyan) [inline keras-dreambooth-sprint/README.md:77]: Repository is more comprehensive.

File: keras-dreambooth-sprint/README.md [hunk 4]
```diff
@@ -1,95 +1,135 @@
...
+Note: maybe let's change the repo_id in push_to_hub_keras to push to the organization.
```
Comment (merveenoyan) [inline keras-dreambooth-sprint/README.md:78]: we left it up to people, that's why we have tags in readmes.

File: keras-dreambooth-sprint/README.md [hunk 5]
```diff
@@ -99,28 +139,29 @@ output = gr.Gallery(label="Outputs").style(grid=(2,2))
 # pass function, input type for prompt, the output for multiple images
 gr.Interface(infer, inputs=["text"], outputs=[output]).launch()
 ```
+Q: should the part above be in the Collab already so it's end-to-end?
```
Comment (merveenoyan) [inline keras-dreambooth-sprint/README.md:142]: no, we already give link to Space and application file is over there with more features.

File: keras-dreambooth-sprint/README.md [hunk 6]
```diff
@@ -135,20 +176,11 @@ tensorflow
...
 
-You can make submission in three themes: 
-
-- Nature and Animals (`nature`)
-- Sci-fi/Fantasy Universes (`sci-fi`)
-- Consentful (`consentful`): Partner up with an artist to fine-tune on their style, with their consent! Make sure to include a reference to the artist’s express consent (e.g. a tweet) in your model card.
-- Wild Card (`wild-card`): If your submission belongs to any category that is not above, feel free to tag it with wild-card so we can evaluate it out of that category.
-
-Add the category with their IDs to the model cards for submission and add `keras-dreambooth` to model card metadata in tags section. Here's an example [model card](https://huggingface.co/spaces/keras-dreambooth/example-submission/blob/main/README.md). All the submissions will be populated [in this leaderboard](https://huggingface.co/spaces/keras-dreambooth/leaderboard) and ranked according to likes on a given Space to determine the winners.
+## Sprint Prizes
```
Comment (merveenoyan) [inline keras-dreambooth-sprint/README.md:181]: why did you remove categories? it's very ambiguous.

File: keras-dreambooth-sprint/README.md [hunk 7]
```diff
@@ -1,95 +1,135 @@
...
+- Consentful (`consentful`): Partner up with an artist to fine-tune on their style, with their consent! Make sure to include a reference to the artist’s express consent (e.g. a tweet) in your model card.
+- Wild Card (`wild-card`): If your submission belongs to any category that is not above, feel free to tag it with wild-card so we can evaluate it out of that category.
 
-[Link to notebook](https://colab.research.google.com/drive/1Bh49YBYgaNc3X5aRzaMZNska4ErvU3lJ?usp=sharing) 
+Add the category with their IDs to the model cards for submission and add `keras-dreambooth` to model card metadata in tags section. Here's an example [model card](https://huggingface.co/spaces/keras-dreambooth/example-submission/blob/main/README.md). All the submissions will be populated [in this leaderboard](https://huggingface.co/spaces/keras-dreambooth/leaderboard) and ranked according to likes on a given Space to determine the winners.
 
-You can fine-tune on any concept that you want. Couple of inspirations for you: 
+Couple of inspirations for you: 
 
+TODO: it might be confusing to show examples not using keras. maybe worth for us to build some of the examples and share them
```
Comment (merveenoyan) [inline keras-dreambooth-sprint/README.md:60]: I think below ones are very cool examples and I'm in favor of keeping them.

File: keras-dreambooth-sprint/README.md [hunk 8]
```diff
@@ -1,95 +1,135 @@
...
+TODO: Add official dates
+
+
+## Getting Started  🚀 
+
+To get started, join us in [hf.co/join/discord](http://hf.co/join/discord) and take the role #open-source, and meet us in #keras-working-group channel. 
+
+We will be hosting our demos in this organization on Hugging Face Hub: [keras-dreambooth](https://huggingface.co/keras-dreambooth), send a request to join [here](https://huggingface.co/organizations/keras-dreambooth/share/RMocthadPgpxxUDHtAesrbBzieDLgUfPmv) if you’d like to make a submission 🙂
+
+## Submission Categories
```
Comment (merveenoyan) [inline keras-dreambooth-sprint/README.md:48]: I think this doesn't make sense without seeing the whole workflow and what people will do, at this line of guide you don't know much about what you will do as a participant, so I'd rather keep it below.

**Review by merveenoyan — COMMENTED** [review]
@osanseviero thanks for the nits 🤗

File: keras-dreambooth-sprint/README.md [hunk 9]
```diff
@@ -1,95 +1,135 @@
...
 
-### Dreambooth Diffusers Integration with KerasCV
+## Model Training
 
-As of now, inference and deployment options of `KerasCV` is limited, which `diffusers` library could come for a rescue. With only few lines of code, we can convert a `KerasCV` model into a `diffusers` one and use `diffusers`’ pipelines to perform inference. You can get more information [here](https://huggingface.co/docs/diffusers/main/en/using-diffusers/kerascv). Also check out [this Space](https://huggingface.co/spaces/sayakpaul/convert-kerascv-sd-diffusers) for converting your `KerasCV` model to `diffusers`one.
+You can find the notebook here and adjust it according to your own dataset 👇  
 
-```
-from diffusers import StableDiffusionPipeline
+TODO: The notebook below is a Google Drive. We should add it to the GitHub repo
```
Comment (sayakpaul) [inline keras-dreambooth-sprint/README.md:73]: I think it's okay for now.

File: keras-dreambooth-sprint/README.md [hunk 10]
```diff
@@ -1,95 +1,135 @@
...
 
 ![Metadata](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/dreambooth-etadata.png)
 
-Then, we will fill the markdown part. Hyperparameters and plot is automatically generated so we can write a short explanation for description, intended use and dataset.
-
+Then, we will fill the markdown part. Hyperparameters and plot are automatically generated so we can write a short explanation for description, intended use and dataset.
 
 You can find the example repository below 👇 
 
 [keras-dreambooth/dreambooth_diffusion_model · Hugging Face](https://huggingface.co/keras-dreambooth/dreambooth_diffusion_model)
```
Comment (sayakpaul) [inline keras-dreambooth-sprint/README.md:94]: I think this name needs to change to `keras-dreambooth/dreambooth_diffusion_model`?

File: keras-dreambooth-sprint/README.md (hunk 6 above)
Comment (sayakpaul) [reply keras-dreambooth-sprint/README.md:181]: I am in favor of keeping the categories.

**Review by sayakpaul — COMMENTED** [review]
Thanks for the changes. Left some comments.

File: keras-dreambooth-sprint/README.md (hunk 2 above)
Comment (osanseviero) [reply keras-dreambooth-sprint/README.md:75]: This code is still in line 109 in the "Dreambooth Diffusers Integration with KerasCV" section

File: keras-dreambooth-sprint/README.md (hunk 4 above)
Comment (osanseviero) [reply keras-dreambooth-sprint/README.md:78]: Ah, that was a bit confusing to me

> We will be hosting our demos in this organization on Hugging Face Hub: [keras-dreambooth](https://huggingface.co/keras-dreambooth), send a request to join [here](https://huggingface.co/organizations/keras-dreambooth/share/RMocthadPgpxxUDHtAesrbBzieDLgUfPmv) if you’d like to make a submission 🙂

So IIUC, users can upload the models to their own accounts but all demos go together. Is that right?

File: keras-dreambooth-sprint/README.md (hunk 5 above)
Comment (osanseviero) [reply keras-dreambooth-sprint/README.md:142]: The expectation of this section is that a user will locally do an `app.py` and then copy paste/drag and drop it to the Space. I was wondering if it does not make sense to have a small gradio demo at the end of the colab so people learn to run a demo in colab, and then copy paste it to Spaces

File: keras-dreambooth-sprint/README.md (hunk 9 above)
Comment (osanseviero) [reply keras-dreambooth-sprint/README.md:73]: I think having it in the repo for the event would be useful, that way others can open suggestions, fix typos, etc. As an example done from previous event: https://github.com/huggingface/community-events/tree/main/whisper-fine-tuning-event

**Comment by osanseviero** [comment]
The diff looks a bit large, but most of the content is the same. Here is a list of the major changes

* Added a Table of Contents
* Added a couple of questions that were unclear to me while going as a newcommer
* Changed `**` to `##` so we can link to them from ToC
* Moved categories section a bit closer to the beginning (can revert that based on the comment, let's discuss over there)
* Moved the diffusers integration part after the Colab part, as exporting to diffusers format is done after training the model

**Comment by merveenoyan** [comment]
If this is okay, I will close this PR and apply the changes myself, as I kinda found this way of reviewing and applying changes (comments in guide itself and sections moving to other places) a bit chaotic and hard to review. You can review my PR instead if this works for you.

File: keras-dreambooth-sprint/README.md [hunk 11]
```diff
@@ -1,95 +1,135 @@
...
+TODO: also convert this repo to diffusers so the API is enabled
+TODO: should we have proper metadata in this repo as well? (e.g. missing the submission or keras event tags)
+TODO: mention which other tags need to be added
```
Comment (sayakpaul) [inline keras-dreambooth-sprint/README.md:97]: @osanseviero 

@merveenoyan and I had a brief discussion on this. IMO, we could just hyperlink https://huggingface.co/docs/diffusers/main/en/using-diffusers/kerascv saying that the participants are free to use `diffusers` after their models have been trained in TensorFlow and Keras. We also add a one-liner to denote why converting to diffusers might be helpful (gives flexibility in using different schedulers, offers optimizations, etc.). 

So, with that in mind, it might be easier to ask the participants to submit Spaces with a disclaimer that the underlying models need to be trained using TensorFlow and Keras (conversion to `diffusers` part being optional).
